### ISSUE TITLE
Elden Ring still not working

### ISSUE THREAD

**Issue by Shahelion** [tuffee88/d3d12ProxyEdrDx11_0#73]
I've followed every step and this stupid white screen is still plaguing me, please help. It does a white screen, then a black screen, then crashes, every time. I can see and move the mouse but that's it.

I have attached the DebugView log and the Event Viewer log.
[DESKTOP-QMKHQ2J.LOG](https://github.com/user-attachments/files/16032536/DESKTOP-QMKHQ2J.LOG)
[EventViewerLog.txt](https://github.com/user-attachments/files/16032534/EventViewerLog.txt)

**Comment by tuffee88** [comment]
Hi @Shahelion,
according to the logfile you get past DX12 init so that's likely something different than what is fixable with this project.
Can you provide some more information about your graphics card ?

**Comment by Shahelion** [comment]
AMD Radeon R7 200 Series

I was able to run the game before the DLC release

**Comment by qleksd** [comment]
I've also done every step and the game still crashes. It stays minimized and when i click on the icon it opens for 1 sec and minmizes again. Using GTX 1650

**Comment by tuffee88** [comment]
@qleksd: With your GTX 1650 you should be able to run the game as-is, this fix is definitedly not required. I'd suggest reinstalling if you have problems.

**Comment by tuffee88** [comment]
@Shahelion: Which GPU model are you using exactly (R7 200 branded cards can be any of four generations of AMD GPU designs so the featureset is quite different - with many of them not needing this fix as the are DX12 FL12_0 native cards) ?

**Comment by Shahelion** [comment]
[DxDiag.txt](https://github.com/user-attachments/files/16654371/DxDiag.txt)

**Comment by qleksd** [comment]
@tuffee88 Yeah, i think found out what was the problem. I probably needed to update my drivers.
Thanks for the reply though

**Comment by Shahelion** [comment]
There are no further updates available for my drivers.
